astro/src/content/docs/lifecycle/examples/authenticate-users-with-saml.mdx

```diff
@@ -0,0 +1,466 @@
+
+You are contacted by a new corporate client, called Corpo.com, which wants all its employees to use your service. However, Corpo requires authentication to be done with SAML, using Corpo as the identity provider (the party that holds the database of users). You want to add SAML authentication to your website without coding it yourself, so you are considering using FusionAuth as your gateway. In this case, FusionAuth will be used only as a service provider (requesting party), not an identity provider.
+
+Let's also consider the possibility that Corpo is happy to allow your website to be the identity provider, as long as SAML is used. In either case, you'll use FusionAuth to manage SAML, hiding the complexity of the protocol from your website.
+
+<Aside type="tip">
+If you use FusionAuth with corporate software, you may also want to provision users automatically when they join or leave your company. Learn [how to do this with SCIM](/docs/lifecycle/examples/manage-user-with-scim).
+</Aside>
+
+## Authentication Flow
+
+Before configuring FusionAuth to support the example scenarios, let's look at how the SAML authentication process works. If you have used OAuth before, the flow is almost identical — it's only the underlying implementation that differs between OAuth and SAML.
+
+First, consider the simple case in which your website, Changebank.com, does its own authentication, as shown below. Only two parties are involved: the user and the website.
+
+<Diagram1></Diagram1>
+
+Next, consider using FusionAuth as the SAML identity provider for login. Instead of letting the user log in on your site, you'll redirect them to FusionAuth to log in, where an identifying cookie will be stored in their browser, and they'll be sent back to your site. The flow below will look familiar to anyone who has used OAuth before.
+
+<Diagram2></Diagram2>
+
+You can read more about using FusionAuth as a SAML identity provider in the [documentation](/docs/lifecycle/authenticate-users/saml).
+
+Finally, consider the Corpo.com primary business request that Corpo acts as the identity provider. In this case, there are four parties: the user, Changebank, FusionAuth acting as a service provider, and Corpo acting as the identity provider. This authentication flow is similar to the flow above, with one more level of redirection.
+
+<Diagram3></Diagram3>
+
+You can read more about using FusionAuth as a service provider to connect to another identity provider in the [documentation](/docs/lifecycle/authenticate-users/identity-providers/overview-samlv2).
+
+In any of these cases, if the user already has an authentication cookie for the website, they will bypass the login page and be redirected to their account page.
+
+## Example 1: Use FusionAuth As A SAML Service Provider
```

Comment:
Let's strike this entire section as it doesn't add much past the Okta setup guide.